pkgs/applications/science/electronics/qucs-s/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, lib, fetchFromGitHub, flex, bison, qt4, libX11, cmake, gperf, adms,
+ngspice, wrapGAppsHook,
+kernels ? [ ngspice ] }:
+
+stdenv.mkDerivation rec {
+  pname = "qucs-s";
+  version = "0.0.22";
+
+  src = fetchFromGitHub {
+    owner = "ra3xdh";
+    repo = "qucs_s";
+    rev = version;
+    sha256 = "0rrq2ddridc09m6fixdmbngn42xmv8cmdf6r8zzn2s98fqib5qd6";
+  };
+
+  nativeBuildInputs = [ wrapGAppsHook cmake ];
+  buildInputs = [ flex bison qt4 libX11 gperf adms ] ++ kernels;
+
+  preConfigure = ''
+    # Make custom kernels avaible from qucs-s
+    gappsWrapperArgs+=(--prefix PATH ":" ${lib.makeBinPath kernels})
+  '';
+
+  enableParallelBuilding = true;
```

Comment:
You don't need it, cmake hook enables parallel building by default, see https://github.com/NixOS/nixpkgs/blob/04150a003ea363f8e365bab1d1b3cae721e3fe2f/pkgs/development/tools/build-managers/cmake/setup-hook.sh#L127-L130

Reply:
Very interesting, thanks for letting me know